source/wp-content/themes/wporg-developer-2023/templates/single-blocks-handbook.html

```diff
@@ -0,0 +1,85 @@
+
+				<!-- wp:paragraph {"fontSize":"small","className":"is-link-to-top"} -->
+				<p class="has-small-font-size is-link-to-top"><a href="#wp--skip-link--target">↑ Back to top</a></p>
+				<!-- /wp:paragraph -->
+			</aside>
+			<!-- /wp:group -->
+
+			<!-- wp:post-content {"layout":{"inherit":true}} /-->
+		</article>
+		<!-- /wp:group -->
+
+		<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|40"},"margin":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"},"blockGap":"var:preset|spacing|20"},"border":{"top":{"color":"var:preset|color|light-grey-1","width":"1px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+		<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--light-grey-1);border-top-width:1px;margin-top:var(--wp--preset--spacing--40);margin-bottom:var(--wp--preset--spacing--40);padding-top:var(--wp--preset--spacing--40)">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
+				<p style="font-style:normal;font-weight:700">First published</p>
+				<!-- /wp:paragraph -->
+		
+				<!-- wp:post-date {"fontSize":"normal"} /-->
+			</div>
+			<!-- /wp:group -->
+		
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
+				<p style="font-style:normal;font-weight:700">Last updated</p>
+				<!-- /wp:paragraph -->
+		
+				<!-- wp:post-date {"displayType":"modified","fontSize":"normal"} /-->
+			</div>
+			<!-- /wp:group -->
+
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
+				<p style="font-style:normal;font-weight:700">Edit article</p>
+				<!-- /wp:paragraph -->
+		
+				<!-- wp:paragraph {"className":"external-link"} -->
+				<p class="external-link"><a href="#">Improve it on GitHub</a></p>
+				<!-- /wp:paragraph -->
+			</div>
+			<!-- /wp:group -->
+
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
+				<p style="font-style:normal;font-weight:700">Changelog</p>
+				<!-- /wp:paragraph -->
+		
+				<!-- wp:paragraph {"className":"external-link"} -->
+				<p class="external-link"><a href="#">See list of changes</a></p>
```

Comment:
Are we going to follow up with these links?

Reply:
Yeah, there's still plenty to follow up with — this section also needs mobile styling, for example. I honestly meant to draft this PR but when I didn't I decided to let it be a first-pass & iterate in trunk 😅 

If you'd rather I keep iterating in the branch though, I can swap this into to draft mode.

Reply:
Nope, merge it in when you think it's ready.